index.html

```diff
@@ -0,0 +1,36 @@
+      
+<h2 >"Dr.Norman Borlaung Tribute"</h2>
+<div class="top-section">
```

Comment:
I kindly suggest that instead of using div tag at this point I think it would be best practice using Section tags for best practice of html sematic tags.